Packs/CheckpointFirewall/Integrations/CheckPointFirewall_v2/CheckPointFirewall_v2.py

```diff
@@ -1786,7 +1810,11 @@ def main():
                 return_results(test_module(base_url, sid, verify_certificate))
                 checkpoint_logout_command(base_url, sid, verify_certificate)
                 return
-
+        elif command == 'checkpoint-mds-login-and-get-session-id':
+            return_results(checkpoint_mds_login_and_get_sid_command(base_url, username, password,
```

Comment:
it seems to me that you can modify the existing `checkpoint_login_and_get_sid_command` function and `checkpoint-login-and-get-session-id` and add non mandatory arg `domain`. 
`domain` parameter will be passed to `checkpoint_login_and_get_sid_command`, if exists then will be used, otherwise the login will happen as previously. 
If domain was provided as an argument to the command itself, then it will override the `domain` from parameters.|
@FFrozTT what do you think?